src/styles/points/points_fragment.glsl

```diff
@@ -49,14 +49,20 @@ void main (void) {
     #ifdef TANGRAM_MULTI_SAMPLER
     if (v_sampler == 0.) { // sprite sampler
     #endif
-        #ifdef TANGRAM_POINT_TEXTURE
+        #ifdef TANGRAM_TEXTURE_POINT
             // Draw sprite
             color *= texture2D(u_texture, v_texcoord);
         #else
             // Draw a point
             vec2 uv = v_texcoord * 2. - 1.; // fade alpha near circle edge
             float point_dist = length(uv);
-            color.a = clamp(color.a - (smoothstep(0., TANGRAM_FADE_RANGE, (point_dist - TANGRAM_FADE_START)) / TANGRAM_FADE_RANGE), 0., color.a);
+            color = mix(
+                color,
+                v_outline_color,
+                (1. - smoothstep(v_outline_edge - v_aa_factor, v_outline_edge + v_aa_factor, 1.-point_dist)) * step(.000001, v_outline_edge)
+            );
+            color.a = mix(color.a, 0., (smoothstep(1. - v_aa_factor, 1., point_dist)));
```

Comment:
Hey, sorry to open this conversation but I don't fully understand this.

`v_aa_factor` is calculated taking like `TANGRAM_PX_FADE_RANGE * 1.0/ length(a_shape.xy / 256.)`. 

This opens some questions:
- even `v_aa_factor` is a varying, does it vary? the v_aa_factor calculated for each vertex should be the same (I'm talking just about the quads used to render a point)
- when point size is <= 2px `v_aa_factor` is greater than 1 so the `smoothstep(1. - v_aa_factor, 1., point_dist)` is kind of misleading
- if `a_shape` is [+-128, +-128], then should n't be `TANGRAM_PX_FADE_RANGE * 1.0/ length(a_shape.xy / 128.)` or something like `max(1.0, TANGRAM_PX_FADE_RANGE * 1.0/ length(a_shape.xy / 256.))` to avoid aliasing on 1px points? 

Am I misunderstanding `v_aa_factor` usage?

thanks!